dev-libs/antlr-c/antlr-c-3.5.2-r2.ebuild

```diff
@@ -19,20 +19,17 @@ IUSE="debug debugger doc"
 BDEPEND="doc? ( app-doc/doxygen[dot] )"
 
 PATCHES=( "${FILESDIR}/3.5-cflags.patch" )
-MULTILIB_WRAPPED_HEADERS=( /usr/include/antlr3config.h )
-DOCS=( AUTHORS ChangeLog NEWS README )
 
 src_prepare() {
 	default
+
 	sed -i '/^QUIET/s/NO/YES/' doxyfile || die
 	eautoreconf
-	multilib_copy_sources
 }
 
-multilib_src_configure() {
+src_configure() {
 	local econfargs=(
 		--enable-shared
```

Comment:
Do you need an explicit `--enable-shared`?

Reply:
No!

```
  --enable-shared[=PKGS]  build shared libraries [default=yes]
```

Dropped.